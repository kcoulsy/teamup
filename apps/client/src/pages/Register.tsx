import React from 'react';

import { Row, Col } from 'antd';
import RegisterForm from '../components/RegisterForm/RegisterForm';

function Register() {
  return (
    <Row justify='center'>
      <Col>
        <RegisterForm />
      </Col>
    </Row>
  );
}

export default Register;
