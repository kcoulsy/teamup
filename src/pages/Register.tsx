import React from 'react';

import RegisterForm from '../components/RegisterForm/RegisterForm';
import { Row, Col } from 'antd';

const Register: React.FunctionComponent = () => {
    return (
        <Row justify="center">
            <Col>
                <RegisterForm />
            </Col>
        </Row>
    );
};

export default Register;
