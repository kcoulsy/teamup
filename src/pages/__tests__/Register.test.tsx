import React from 'react';
import { shallow } from 'enzyme';
import { describe, expect, test } from 'vitest';
import Register from '../Register';
import { shallowToJson } from 'enzyme-to-json';

describe('render Register component', () => {
  test('should render Register page component', () => {
    const wrapper = shallow(<Register />);

    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
