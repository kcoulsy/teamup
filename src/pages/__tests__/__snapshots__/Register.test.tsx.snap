// Vitest Snapshot v1

exports[`render Register component > should render Register page component 1`] = `
<Row
  justify="center"
>
  <Col>
    <Connect(RegisterForm) />
  </Col>
</Row>
`;

exports[`render Register component should render Register page component 1`] = `
<div>
  <h3>
    Register Page
  </h3>
  <Connect(Component) />
</div>
`;
